### ISSUE TITLE
buddy-sign 0.10.0 includes user.clj

### ISSUE THREAD

**Issue by codahale** [funcool/buddy-sign#20]
I just upgraded buddy-sign to 0.10.0 and noticed it broke my project’s uberjars. It turns out your release process accidentally included your `user.clj` in the 0.10.0 JAR:

```clojure
(ns user
  (:require
   [clojure.java.io :as io]
   [clj-time.core :as t]
   [buddy.sign.jws :as jws]
   [buddy.core.keys :as keys]))

;; (def ec-privkey (keys/private-key "test/_files/privkey.ecdsa.pem" "secret"))
;; (def ec-pubkey (keys/public-key "test/_files/pubkey.ecdsa.pem"))

(defn- private-key []
  (keys/private-key
   (io/resource "_files/privkey.ecdsa.pem") "secret"))

(defn- public-key []
  (keys/public-key
   (io/resource "_files/pubkey.ecdsa.pem")))

(defn new-token
  [res]
  (let [exp (t/plus (t/now) (t/days 1))]
    (jws/sign res (private-key) {:alg :es512 :exp exp})))

(defn from-token
  [token]
  (try
    (jws/unsign token (public-key))
    (catch Exception e
      (do (prn e)
          nil
          ))))

(comment
  (def token (new-token {:foo "bar"}))
  (jws/unsign token (pubkey))
  )
```

As a result, my uberjars don’t contain `clj-time` because it’s already loaded by the `user` ns.

**Comment by niwinz** [comment]
Released 0.11.0 without the `user.clj` sorry for inconvenience.

**Comment by codahale** [comment]
0.11.0 fixes my problem! Thank you for both the prompt response *and* the excellent set of libraries!
